src/test/crossFetchNode.test.ts

```diff
@@ -165,14 +158,12 @@ test('should not fail when arbitrary parameters are configured in fetchOptions',
     .matchHeader('authorization', 'Bearer test-auth')
     .reply(200, {}, { 'content-type': 'application-json charset=UTF-8' });
 
-  const clientConfig = {
+  const clientConfig: ClientConfigInit = {
     ...config,
-    ...{
-      fetchOptions: {
-        somekey: 'some value',
-        timeout: 1000,
-      },
-    },
+    fetchOptions: {
+      somekey: 'some value',
+      timeout: 1000,
+    } as any,
```

Comment:
why as any?

Reply:
Our type definition doesn't allow arbitrary parameters, but that's exactly what this is testing. Using the type assertion `as any` is a workaround.

On a broader note, all we do is pass the options through to `fetch`, unmodified. I don't know what the rationale is for why this test exists. It seems to only be testing the third party dependency, rather than our own code.